app/components/navbar.js

```diff
@@ -33,6 +34,20 @@ export default class NavbarComponent extends Component {
     },
   ];
 
+  @action async logoutHandler() {
+    try {
+      const response = await fetch(LOGOUT_URL, {
+        method: 'GET',
+        credentials: 'include',
+      });
+      if (response.status === 200) {
+        location.reload();
```

Comment:
Instead of reloading, shouldn't we need to redirect the user to `realdevsquad.com` as my-site is kind of user's dashboard and we requires login to GitHub.